Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "cexpr"
-version = "0.4.0"
+version = "0.5.0"
```

Comment:
The API didn't change in the nom update did it? If it did nevermind, but otherwise perhaps it should be 0.4.1?

Reply:
That's my understanding as well. I followed 7a88526 which bumped the minor version following the nom update. But I agree, increasing the patch version makes more sense.

Reply:
I think this crate uses a `nom` type in its public API.

Reply:
You are right, it does [re-export some types](https://github.com/jethrogb/rust-cexpr/blob/master/src/lib.rs#L22). I don't know if some of these have changed since the previous version.

Either way, since we are still at major version zero (0.x), it doesn't matter. Please make a decision on what you would prefer and I'll adapt this request accordingly. Thanks.

Reply:
It does matter. Since we re-export those types, every time we update the dependency “major” version, that's also a change in our public API, so we need to update our own “major” version to avoid downstream breakage. In the context of cargo semver, “major” here means the first digit for 1.0.0 and up, and the first and second digits for below 1.0.0.

Reply:
I wasn't aware of Cargo's spin on semver for version 0, thanks for sharing this. I can see that a least one variant was added to the ErrorKind type ([5.x](https://docs.rs/nom/5.1.2/nom/error/enum.ErrorKind.html) vs [6.x](https://docs.rs/nom/6.1.2/nom/error/enum.ErrorKind.html)). So we should bump to `0.5.0`. Do you agree?

Reply:
It doesn't matter if any variants were added. nom 5.x is a different crate than nom 6.x, so the types are different, so as mentioned in my previous post
> every time we update the dependency “major” version, that's also a change in our public API, so we need to update our own “major” version to avoid downstream breakage.

Reply:
Done